eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -93,6 +118,31 @@ jobs:
       - script: |
           node common/scripts/install-run-rush.js build:test -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build test assets"
+        condition: and(succeededOrFailed(),eq(variables['DependencyVersion'],''))
+      
+      - pwsh: |
+          npm install
+          node index.js --artifact-name "${{parameters.PackageName}}" --version-type $(DependencyVersion) --source-dir "$(Build.SourcesDirectory)" --test-folder "test/public"
+        displayName: "Convert tests to run against package references"
+        workingDirectory: $(Build.SourcesDirectory)/eng/tools/dependency-testing
+        condition: and(succeededOrFailed(),ne(variables['DependencyVersion'],''))
+
+      - pwsh: |
+          node common/scripts/install-run-rush.js update
+        displayName: "Rush update"
+        workingDirectory: $(Build.SourcesDirectory)
+        condition: and(succeededOrFailed(),ne(variables['DependencyVersion'],''))
+
+      - pwsh: |
+          node ../../../../../common/scripts/install-run-rushx.js build
```

Comment:
Once the relative paths reach this point it is probably better to use a variable for the absolute path, like `$(Build.SourceDirectory)`